amd_openvx/openvx/ago/ago_haf_cpu_geometric.cpp

```diff
@@ -2434,6 +2450,28 @@ vx_uint32     srcImageStrideInBytes
 			_mm_store_si128(dstNext++, pixels2);
 			_mm_store_si128(dstNext++, pixels1);
 		}
+
+		if (fallbackToC)
+		{
+			unsigned char *src = (unsigned char *) pSrcImage;
+			unsigned char *dst = (unsigned char *) pchDst;
+			unsigned char *dstNext = (unsigned char *) (pchDst + dstImageStrideInBytes);
+			unsigned char *dstLast = dst + dstWidth - 1;
+
+			while (dst < dstLast)
+			{
```

Comment:
please change to *dst++ = src and remove next line for fewer lines of code

Reply:
I've created pull request with needed changes at https://github.com/kiritigowda/MIVisionX/pull/1

Reply:
I have merged the PR from @milkoleporis on personal branch. I will merge this into MIVisionX.

Thanks all!